gitlab/__init__.py

```diff
@@ -19,17 +19,18 @@
 import warnings
 from typing import Any
 
-import gitlab.config  # noqa: F401
-from gitlab.__version__ import (  # noqa: F401
-    __author__,
-    __copyright__,
-    __email__,
-    __license__,
-    __title__,
-    __version__,
-)
-from gitlab.client import Gitlab, GitlabList  # noqa: F401
-from gitlab.exceptions import *  # noqa: F401,F403
+from . import config as config  # noqa: F401
+from . import const as const
+from . import exceptions as exceptions  # noqa: F401
+from .__version__ import __author__ as __author__  # noqa: F401
+from .__version__ import __copyright__ as __copyright__  # noqa: F401
+from .__version__ import __email__ as __email__  # noqa: F401
+from .__version__ import __license__ as __license__  # noqa: F401
+from .__version__ import __title__ as __title__  # noqa: F401
+from .__version__ import __version__ as __version__  # noqa: F401
```

Comment:
off-topic comment first. I would like to rename `__version__.py` to `version.py` as it is confusing that we import `__version__` but then change `__version__` to become `__version__`. Do you understand????? 😕

Reply:
> Maybe we can just move them back in here to avoid having to do all this? `__init__.py` is quite small now. If needed we can keep only `__version__` there. Although I do see many projects just import these from `__version__` without re-exporting, maybe because it's mostly just used for packaging.

Sorry. I don't quite understand. Move what back in here? I'm not sure what "them" is in "just move them back in here".

Reply:
Trying to remember why I picked `__version__.py`, I think it was a popular convention I saw in repos like https://github.com/psf/requests/blob/main/requests/__version__.py. `_version.py` seemed to be another alternative mostly generated from `setuptools-scm`.

> Sorry. I don't quite understand. Move what back in here? I'm not sure what "them" is in "just move them back in here".

Sorry I wasn't very clear. I mean just moving all those dunder variables from an extra file back into `__init__.py`. They were originally in `__init__.py` and I moved them out for clarity and the release script, but now that the client code has moved out of `__init__.py` it's maybe completely ok to have them back in there, not sure.

Reply:
> > Sorry. I don't quite understand. Move what back in here? I'm not sure what "them" is in "just move them back in here".
> 
> Sorry I wasn't very clear. I mean just moving all those dunder variables from an extra file back into `__init__.py`. They were originally in `__init__.py` and I moved them out for clarity and the release script, but now that the client code has moved out of `__init__.py` it's maybe completely ok to have them back in there, not sure.

If you mean just move everything in `__version__.py` into `__init__.py` that sounds good to me. Simplifies things in my mind. If you meant something else, then I'm still confused 😅

Reply:
That's what I meant yes haha :grinning: at least, if we can. I think the version variable gets imported somewhere else.

Reply:
I don't think it will work 😢 Due to circular imports... I tried and ran into that issue.

Reply:
I went with creating a new 1st commit that renames `__version__.py` to `version.py`. Due to the issues with circular import. For almost everything I used the top-level namespace to get the value. `gitlab/const.py` was the outlier that caused the circular import so I imported from `version.py` for it.